src/Collapsable.jsx

```diff
@@ -0,0 +1,99 @@
+var React = require('react');
+var joinClasses = require('./utils/joinClasses');
+var BootstrapMixin = require('./BootstrapMixin');
+var CollapsableMixin = require('./CollapsableMixin');
+var classSet = require('./utils/classSet');
+var domUtils = require('./utils/domUtils');
+var cloneWithProps = require('./utils/cloneWithProps');
+
+var ValidComponentChildren = require('./utils/ValidComponentChildren');
+var createChainedFunction = require('./utils/createChainedFunction');
+
+
+var Collapsable = React.createClass({
```

Comment:
You misspelled Collapsible, alsothis is specific to `Nav`s, so can we name it `CollapsibleNav`?